uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -703,6 +710,35 @@
           it('does not contain admin in the list of managees', function () {
             expect(_.find(controller.managedContacts, { 'id': adminId })).toBeUndefined();
           });
+
+          describe('on absence period change', function () {
```

Comment:
why is this in the "admin opens" section? This is not something that happens only for admins

Reply:
Apologies. Done.